Security/Standard_Levels.mediawiki

```diff
@@ -55,296 +38,128 @@ Changes are detailed in the [https://github.com/mozilla/wikimo_opsec/commits/mas
+== Standard risk levels definition and nomenclature ==
+''The risk levels also represent a simplified ISO 31000 equivalent (and are non-compliant) .''
+These levels are also used to display risk impact, risk probability and any risk related level.
 
 {| class="wikitable"
 |-
-! Level
-! Time spent
-|-
-! <span style="background-color: #cccccc; border-radius: .25em; color: #000000; display: inline-block; font-weight: bold; margin: .1em 0; min-width: 6em; padding: .05em .5em; text-transform: uppercase; text-align: center;">LOW</span>
-|
-* Small group: Few hours.
-* Large group: Few minutes.
-
-Example: Flip a configuration switch, change a password, lookup a document, etc.
-|-
-! <span style="background-color: #4a6785; border-radius: .25em; color: #ffffff; display: inline-block; font-weight: bold; margin: .1em 0; min-width: 6em; padding: .05em .5em; text-transform: uppercase; text-align: center;">MEDIUM</span>
-|
-* Small group: A few days.
-* Large group: A few hours.
-
-Example: Take a group decision, create standards, lookup complex data, manual upgrades, etc.
-|-
-! <span style="background-color: #ffd351; border-radius: .25em; color: #594300; display: inline-block; font-weight: bold; margin: .1em 0; min-width: 6em; padding: .05em .5em; text-transform: uppercase; text-align: center;">HIGH</span>
-|
-* Small group: A week or two.
-* Large group: A day or two.
-
-Example: Implement a new security control, code a new feature, change all company user passwords, etc.
-|-
-! <span style="background-color: #d04437; border-radius: .25em; color: #ffffff; display: inline-block; font-weight: bold; margin: .1em 0; min-width: 6em; padding: .05em .5em; text-transform: uppercase; text-align: center;">MAXIMUM</span>
-|
-* Small group: Weeks, quarterly goals.
-* Large group: Weeks, quarterly goals.
-
-Example: Implement a new security design/change product design, etc.
+! Risk Level
+! Expectations
+! Rationale
 |-
-|}
-
-== RFC2119 handling recommendation levels ==
-See also [https://www.ietf.org/rfc/rfc2119.txt RFC 2119] for a formal definition.
+! <span style="background-color: #d04437; border-radius: .25em; color: #ffffff; display: inline-block; font-weight: bold; margin: .1em 0; min-width: 6em; padding: .05em .5em; text-transform: uppercase; text-align: center;">MAXIMUM Risk</span>
+| ''This is the most important level, where the risk is especially great.''
 
-{| class="wikitable"
-|-
-! Level
-! Coding rationale
-! Definition & expectations
-|-
-|-
-! <span style="background-color: #cccccc; border-radius: .25em; color: #000000; display: inline-block; font-weight: bold; margin: .1em 0; min-width: 6em; padding: .05em .5em; text-transform: uppercase; text-align: center;">OPTIONAL</span>
+* '''Attention''':  Full attention from all concerned parties required.
+* '''Impact''': High or maximum impact.
+* '''Effort''': Greatest amount of effort is required. Following standard/guidelines is required.
```

Comment:
I'm not sure on this attribute. I think what we want to say is, at this level it should require the greatest amount of focus / attention. Is this already captured with the "Attention" bullet?

It reads as though somehow the amount of effort required to resolve something contributes to the risk level, or being at that given risk level.